airbyte-cdk/bulk/core/extract/src/main/kotlin/io/airbyte/cdk/command/SourceConfiguration.kt

```diff
@@ -10,6 +10,9 @@ interface SourceConfiguration : Configuration, SshTunnelConfiguration {
     /** Does READ generate states of type GLOBAL? */
     val global: Boolean
 
+    /** Maximum amount of time may be set to limit overall snapshotting duration */
+    val maxSnapshotReadTime: Duration?
```

Comment:
Rename this to `maxSnapshotReadDuration`? "time" is an ambiguous word

Reply:
Maybe this field should go to `CdcSourceConfiguration`

Reply:
Yeah that's a good rule of thumb

Reply:
changed

Reply:
> Maybe this field should go to CdcSourceConfiguration
the problem is that while this is all CDC related, it's actually on JDBC snapshot partition where we need to deal with this duration limit.
Because jdbc partition reader doesn't know about CdcSourceConfiguration (or about CDC at all) then I had to go to source configuration.

Reply:
this is fine